src/apps/lwaftr/binding_table.lua

```diff
@@ -245,8 +201,7 @@ function BindingTable:lookup_psid(ipv4, port)
 end
 
 function BindingTable:get_br_address(i)
-   assert(i<self.br_address_count)
-   return self.br_addresses[i].addr
+   return self.br_addresses[i+1]
```

Comment:
Both callers look like i will always be valid, though I haven't dug deeply into that.